ts/LinearProgress/index.tsx

```diff
@@ -31,8 +31,8 @@ export class LinearProgress extends MaterialComponent<
     this.afterComponentDidMount();
   }
 
-  public componentWillUpdate(nextProps: ILinearProgressProps) {
-    super.componentWillUpdate(nextProps);
+  public componentWillReceiveProps(nextProps: ILinearProgressProps) {
```

Comment:
add a todo to to switch away from `cWRP` as it is now deprecated

Reply:
2.0